### PR TITLE
input checked

### PR DESCRIPTION
**Pull request by Doriel1121**

### CODE REVIEW

File: souto-app/debug.log [hunk 1]
```diff
@@ -0,0 +1 @@
+[1103/161208.587:ERROR:directory_reader_win.cc(43)] FindFirstFile: The system cannot find the path specified. (0x3)
```
Comment (omer-dayan) [inline souto-app/debug.log]: Delete this file

File: souto-app/src/components/pages/loginPage.js [hunk 2]
```diff
@@ -1,154 +1,154 @@
...
-import axios from 'axios'
-import config from '../../config'
-import { Redirect } from 'react-router-dom'
-import { autoPlay } from 'react-swipeable-views-utils'
-import loginImage0 from '../../resources/login-image-0.jpg'
-import loginImage1 from '../../resources/login-image-1.jpg'
-import loginImage2 from '../../resources/login-image-2.jpg'
-const AutoPlaySwipeableViews = autoPlay(SwipeableViews)
+import React, { Component } from "react";
+import { Button, TextField, Grid, Typography, Input } from "@material-ui/core";
```
Comment (omer-dayan) [inline souto-app/src/components/pages/loginPage.js]: Change your formatter to single quote

Comment (omer-dayan) [reply souto-app/src/components/pages/loginPage.js]: And no ; at the end of line

File: souto-app/src/components/pages/loginPage.js [hunk 3]
```diff
@@ -214,16 +214,19 @@ export default class LoginPage extends Component {
...
                 <Grid item xs={12}>
-                  <Typography variant={'h6'} style={styles.text}>
+                  <Typography variant={"h6"} style={styles.text}>
                     What is your board name?
                   </Typography>
                 </Grid>
                 <Grid item xs={12}>
                   <TextField
+                    inputProps={{
+                      maxLength: 20,
```
Comment (omer-dayan) [inline souto-app/src/components/pages/loginPage.js]: 20 is pretty small no?
"New employee at facebook" is 21 chars. I think 40 is enough

File: souto-app/src/components/pages/loginPage.js [hunk 4]
```diff
@@ -274,17 +277,22 @@ export default class LoginPage extends Component {
...
-                  <Typography variant={'h6'} style={styles.text}>
+                  <Typography variant={"h6"} style={styles.text}>
                     What is your board secret?
                   </Typography>
                 </Grid>
                 <Grid item xs={12}>
-                  <Input
+                  <TextField
+                    onInput={(e) => {
+                      e.target.value = Math.max(0, parseInt(e.target.value))
```
Comment (omer-dayan) [inline souto-app/src/components/pages/loginPage.js:287]: What is this?

File: souto-app/src/components/pages/loginPage.js [hunk 5]
```diff
@@ -420,17 +428,22 @@ export default class LoginPage extends Component {
...
-                  <Typography variant={'h6'} style={styles.text}>
+                  <Typography variant={"h6"} style={styles.text}>
                     What is your board key?
                   </Typography>
                 </Grid>
                 <Grid item xs={12}>
-                  <Input
+                  <TextField
+                    onInput={(e) => {
+                      e.target.value = Math.max(0, parseInt(e.target.value))
```
Comment (omer-dayan) [inline souto-app/src/components/pages/loginPage.js:438]: And here?

File: souto-app/src/components/pages/loginPage.js [hunk 6]
```diff
@@ -470,16 +483,19 @@ export default class LoginPage extends Component {
...
                 <Grid item xs={12}>
-                  <Typography variant={'h6'} style={styles.text}>
+                  <Typography variant={"h6"} style={styles.text}>
                     What is your name?
                   </Typography>
                 </Grid>
                 <Grid item xs={12}>
                   <TextField
+                    inputProps={{
+                      maxLength: 30,
```
Comment (omer-dayan) [inline souto-app/src/components/pages/loginPage.js:493]: Perfect!

File: souto-app/src/components/ticket.js [hunk 7]
```diff
@@ -69,25 +69,26 @@ export default class Ticket extends Component {
             />
           </Grid>
           <Grid
+            zeroMinWidth
```
Comment (omer-dayan) [inline souto-app/src/components/ticket.js]: What is this?

File: souto-app/src/components/ticket.js [hunk 8]
```diff
@@ -69,25 +69,26 @@ export default class Ticket extends Component {
...
             item
             xs={this.props.isManager ? 8 : 10}
             style={styles.textBox}
             onClick={() => {
-              this.setState({ isOpen: true })
+              this.setState({ isOpen: true });
             }}
           >
-            <Typography style={styles.title}>
```
Comment (omer-dayan) [inline souto-app/src/components/ticket.js]: And this?

File: souto-app/src/components/ticket.js [hunk 9]
```diff
@@ -69,25 +69,26 @@ export default class Ticket extends Component {
...
+            <Typography maxLength="2" style={styles.title}>
               {this.props.info.title}
             </Typography>
-            <Typography style={styles.rearInfo}>
+            <Typography noWrap style={styles.rearInfo}>
```
Comment (omer-dayan) [inline souto-app/src/components/ticket.js]: And this?

File: souto-app/src/components/ticketDescription.js [hunk 10]
```diff
@@ -144,8 +144,11 @@ export default class TicketDescription extends Component {
                 variant="outlined"
                 value={this.state.title}
                 fullWidth
+                inputProps={{
+                  maxLength: 20,
```
Comment (omer-dayan) [inline souto-app/src/components/ticketDescription.js]: 20 is not enough. Maybe 50?

File: souto-app/src/components/ticketDescription.js [hunk 11]
```diff
@@ -157,17 +160,20 @@ export default class TicketDescription extends Component {
...
                 fullWidth
                 onChange={(event) => {
-                  this.setState({ description: event.target.value })
+                  this.setState({ description: event.target.value });
                 }}
                 multiline
                 rows={6}
                 rowsMax={6}
+                inputProps={{
+                  maxLength: 80,
```
Comment (omer-dayan) [inline souto-app/src/components/ticketDescription.js]: Here the user suppose to write a description. 80 chars is very samll. Chane it to 300

File: souto-app/src/components/ticket.js (hunk 9 above)
Comment (Doriel1121) [reply souto-app/src/components/ticket.js]: its neccessery so the content of the typography wont exceed the ticket borders

**Pull request merged**